documentation/api/definitions.md

```diff
@@ -116,6 +116,8 @@ Object describing a captured parameter.
 | `value` | string | The parameter value. |
 | `typeName` | string | The parameter type name. |
 | `moduleName` | string | The parameter type module name. |
+| `evalFailReason` | string | The reason why evaluation failed. If missing the evaluation was successful. |
+| `isNull` | bool | Whether the evaluated parameter is null (value is omitted if false). |
```

Comment:
It seems like it might make more sense to remove this field and simply make the `value` field a nullable-string instead. If the value is null, then we use null literal to represent it.

Reply:
That makes sense. I've updated the code to make the parameter value and activityId nullable strings. Nullable types are not enabled on the project level, so I had to enable them in the files I touched and I also had to include a workaround for the `required` modifier in .NET 6